Source/DafnyCore/Options/DafnyProject.cs

```diff
@@ -19,8 +19,6 @@ public class DafnyProject : IEquatable<DafnyProject> {
 
   public string ProjectName => Uri.ToString();
 
-  public bool IsImplicitProject { get; set; }
```

Comment:
What was this deletion about, by the way?

Reply:
It makes it so that the IDE doesn't differentiate between implicit and explicit projects. Previously only implicit projects had features depending on the VerificationTree turned on